src/FixedDataTable.js

```diff
@@ -807,13 +815,19 @@ var FixedDataTable = createReactClass({
     if (this.props.keyboardPageEnabled || this.props.keyboardScrollEnabled) {
       tabIndex = 0
     }
+    var ariaRowCount = this.props.rowsCount + 1;
```

Comment:
You should move this calculation to `_calculateAriaAttributes` as you did for others too (I think it already calculates?)

Reply:
Yep just missed removing it.